### ISSUE TITLE
Remove `allows_sms` and `sms_id_string` fields from `logger_xform` table

### ISSUE THREAD

**Issue by noliveleger** [kobotoolbox/kobocat#639]
With next version of KoBoCAT, SMS support is dropped. These fields are no longer needed. 
Because PostgreSQL needs to rewrite all records in the table to remove those columns, we need to upgrade to PostgreSQL 12 before, to speed-up the migration on big servers.

Blocked by https://github.com/kobotoolbox/tasks/issues/373 and https://github.com/kobotoolbox/kobo-docker/issues/270 (PostgreSQL)

**Comment by jnm** [comment]
Fixed by #692
